worlds/stardew_valley/bundles/bundles.py

```diff
@@ -2,79 +2,95 @@
-    vault = vault_thematic.create_bundle_room(options.bundle_price, random, options)
-    abandoned_joja_mart = abandoned_joja_mart_thematic.create_bundle_room(options.bundle_price, random, options)
-    return [pantry, crafts_room, fish_tank, boiler_room, bulletin_board, vault, abandoned_joja_mart]
-
-
-def get_remixed_bundles(random: Random, options: StardewValleyOptions) -> List[BundleRoom]:
-    pantry = pantry_remixed.create_bundle_room(options.bundle_price, random, options)
-    crafts_room = crafts_room_remixed.create_bundle_room(options.bundle_price, random, options)
-    fish_tank = fish_tank_remixed.create_bundle_room(options.bundle_price, random, options)
-    boiler_room = boiler_room_remixed.create_bundle_room(options.bundle_price, random, options)
-    bulletin_board = bulletin_board_remixed.create_bundle_room(options.bundle_price, random, options)
-    vault = vault_remixed.create_bundle_room(options.bundle_price, random, options)
-    abandoned_joja_mart = abandoned_joja_mart_remixed.create_bundle_room(options.bundle_price, random, options)
-    return [pantry, crafts_room, fish_tank, boiler_room, bulletin_board, vault, abandoned_joja_mart]
-
-
-def get_shuffled_bundles(random: Random, logic: StardewLogic, options: StardewValleyOptions) -> List[BundleRoom]:
-    valid_bundle_items = [bundle_item for bundle_item in all_bundle_items_except_money if bundle_item.can_appear(options)]
-
-    rooms = [room for room in get_remixed_bundles(random, options) if room.name != "Vault"]
+def get_vanilla_bundles(random: Random, content: StardewContent, options: StardewValleyOptions) -> List[BundleRoom]:
+    pantry = pantry_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    crafts_room = crafts_room_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    fish_tank = fish_tank_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    boiler_room = boiler_room_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    bulletin_board = bulletin_board_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    vault = vault_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    abandoned_joja_mart = abandoned_joja_mart_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    raccoon = raccoon_vanilla.create_bundle_room(options.bundle_price, random, content, options)
+    fix_raccoon_bundle_names(raccoon)
+    return [pantry, crafts_room, fish_tank, boiler_room, bulletin_board, vault, abandoned_joja_mart, raccoon]
+
+
+def get_thematic_bundles(random: Random, content: StardewContent, options: StardewValleyOptions) -> List[BundleRoom]:
+    pantry = pantry_thematic.create_bundle_room(options.bundle_price, random, content, options)
+    crafts_room = crafts_room_thematic.create_bundle_room(options.bundle_price, random, content, options)

```

Comment:
There's probably some weird python thing you can do to reduce `get_vanilla_bundles`, `get_thematic_bundles`, and `get_remixed_bundles` into something that duplicates less code. Maybe not, and maybe not worth looking into.